buckwheat/tokenizer.py

```diff
@@ -3,23 +3,24 @@
 """
 import logging
 import os
+from collections import deque
 from tempfile import TemporaryDirectory
 from typing import Iterator, List, Optional, Set, Tuple, Union
 
+import pygments
```

Comment:
order of imports is not alphabetic - please fix it

Reply:
It was because PyCharm imports optimiser by deafault separately orders `import ...` and `from ... import ...`